main.yml

```diff
@@ -604,6 +605,21 @@ rhel-sw:
       - id: x86
         title: x86
 
+ros:
+  title: Resource Optimization
+  api:
```

Comment:
You don't have to include the api part at all if it's not ready. The API team should share api spec on `/${YOUR_APP_NAME}/${API_VERSION}/openapi.json` to allow customers to work with it. If it's not ready yet, you don't have to add it here and wait for it to be ready, once that you can open new PR to include it.

Reply:
Great! As per a suggestion, removed api section. Thanks @karelhala.